main.go

```diff
@@ -312,6 +323,23 @@ func main() {
 		}
 	}()
 
+	if tool.Options.ComplianceAPIURL != "" {
+		wg.Add(1)
+
+		go func() {
+			err := statussync.StartComplianceEventsSyncer(
+				mgrCtx, tool.Options.ClusterNamespace, hubCfg, managedCfg, tool.Options.ComplianceAPIURL, queue,
```

Comment:
The hub certs can rotate/change.  A couple test scenarios are:
1. Make sure we run and go through a cert rotation ok.
2. Make sure we handle or at least understand how to recover if the customer changes their kube api server certs.